content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (Aus-NZ region)"
+date: 2022-03-02
```

Comment:
This date is now in the past

Reply:
@Debanitrkl lemme know what date you want to publish this.

Reply:
Updated the date